docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,859 @@
+    "3. The optimizer class (`QuantumRandomAccessOptimizer`): This class performs the high-level optimization algorithm, utilizing the capabilities of the encoding class and the rounding scheme.\n",
+    "\n",
+    "\n",
+    "### References\n",
+    "\n",
+    "[1] Bryce Fuller et al., *Approximate Solutions of Combinatorial Problems via Quantum Relaxations,*  [arXiv:2111.03167](https://arxiv.org/abs/2111.03167)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_optimization.algorithms.qrao import (\n",
+    "    QuantumRandomAccessEncoding,\n",
+    "    SemideterministicRounding,\n",
+    "    QuantumRandomAccessOptimizer,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up a combinatorial optimization problem\n",
+    "\n",
+    "In this tutorial, we will consider a random max-cut problem instance and use QRAO to try to find a maximum cut; in other words, a partition of the graph's vertices (nodes) into two sets that maximizes the number of edges between the sets.\n",
+    "\n",
+    "To begin, we utilize the `Maxcut` class from Qiskit Optimization's application module. It allows us to generate a `QuadraticProgram` representation of the given graph.\n",
+    "\n",
+    "Note that once our problem is represented as a `QuadraticProgram`, it needs to be converted into the appropriate format for QRAO, which is a [quadratic unconstrained binary optimization (QUBO)](https://en.wikipedia.org/wiki/Quadratic_unconstrained_binary_optimization) problem. While a `QuadraticProgram` generated by `Maxcut` is already in QUBO form, if you define your own problem, make sure to convert it into a QUBO before proceeding. You can refer to this [tutorial](https://qiskit.org/documentation/optimization/tutorials/02_converters_for_quadratic_programs.html) for guidance on converting QuadraticPrograms.\n",
+    "\n",
+    "Note that once our problem has been represented as a `QuadraticProgram`, it will need to be converted to the correct type, a [quadratic unconstrained binary optimization (QUBO)](https://en.wikipedia.org/wiki/Quadratic_unconstrained_binary_optimization) problem, so that it is compatible with QRAO.\n",
+    "A `QuadraticProgram` generated by `Maxcut`is already a QUBO, but if you define your own problem be sure you convert it to a QUBO before proceeding. Here is [a tutorial](https://qiskit.org/documentation/optimization/tutorials/02_converters_for_quadratic_programs.html) on converting `QuadraticPrograms`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\\ This file has been generated by DOcplex\n",
```

Comment:
Could you rerun the script? The output looks the `export_lp_string` format. Maybe the other `prettyprint` part in the bottom too.